### ISSUE TITLE
Possible issue with calculating loss

### ISSUE THREAD

**Issue by zappybiby** [iinc/acc#1]
Looking at this line: https://github.com/iinc/acc/blob/fae904d11ff3e1d08cffe5b4ddee07e62ff418f1/cnn.py#L68
This current implementation does not use the 'onehot_labels' variable that was declared just before it.

I think this should be changed to `loss = tf.losses.softmax_cross_entropy(onehot_labels=onehot_labels, logits=logits)`

This change would make the code consistent with how the code appears in this TensorFlow example:
https://www.tensorflow.org/tutorials/layers#calculate_loss
